hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -137,6 +137,103 @@ public Job createSubmittableJob(Configuration conf) throws IOException {
     return job;
   }
 
+  /**
+   * Sets up the actual job.
+   *
+   * @param conf  The current configuration.
+   * @param args  The command line parameters.
+   * @return The newly created job.
+   * @throws IOException When setting up the job fails.
+   * @deprecated please use main method instead.
```

Comment:
We might want to suggest `This will be deprecated in 3.0 release`.

Reply:
Please also include since which release it is deprecated. We had this case just now that we had a deprecation comment, which was not in line with our ref documentation.